.gitignore

```diff
@@ -10,3 +10,4 @@
 /pkg
 *.exe
 *.iml
+sg_config.json
```

Comment:
Is this something we want in the main repo? Seems a little bit specific...

Reply:
Discussed this with Adam yesterday. Ben and I basically have this as our SG config and I may have accidentally pushed it. This was added to avoid this.